boards/arm/qemu_cortex_m4f/qemu_cortex_m4f_defconfig

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F4X=y
+CONFIG_SOC_STM32F405XG=y
+CONFIG_QEMU_ICOUNT_SHIFT=8
+
+# 8MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=8000000
+
+# Enable clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Enable FPU
+CONFIG_FPU=y
```

Comment:
Probably not needed, since the respective FP tests enable this, but i guess there's no harm.